composer.json

```diff
@@ -1,14 +1,20 @@
 {
-  "name": "nextcloud/user_retention",
-  "description": "user_retention",
-  "license": "AGPL",
-  "config": {
-    "optimize-autoloader": true,
-    "classmap-authoritative": true
-  },
-  "require": {
-  },
-  "scripts": {
-    "lint": "find . -name \\*.php -not -path './vendor/*' -print0 | xargs -0 -n1 php -l"
-  }
+	"name": "nextcloud/user_retention",
+	"description": "user_retention",
+	"license": "AGPL",
+	"config": {
+		"optimize-autoloader": true,
+		"classmap-authoritative": true,
+		"platform": {
+			"php": "7.4"
+		},
+		"sort-packages": true
```

Comment:
For my knowledge, what does `"sort-packages": true` do?

Reply:
Content of `require(-dev)` and other elements is sorted alphabetically when the file is written by composer (so to satisfy my personal needs)